ecl/hql/hqlutil.cpp

```diff
@@ -7381,7 +7381,7 @@ class GccCppNameMangler
             ITypeInfo *paramType = param->queryType();
 
             bool isOut = param->hasAttribute(outAtom);
-            bool isConst = param->hasAttribute(constAtom);
+            bool isConst = !param->hasAttribute(noConstAtom);
```

Comment:
Check for constAtom has been replaced with check for noConstAtom.  It looks like constAtom is still valid - isn't it necessary to check for constAtom?